src/main/java/com/expedia/pitchfork/systems/datadog/forwarder/DatadogSpansDispatcher.java

```diff
@@ -36,13 +38,18 @@
     private final WebClient datadogClient;
     private final ObjectWriter mapper;
     private final ArrayBlockingQueue<Span> pending;
+    private final Counter successCounter;
+    private final Counter failureCounter;
 
     public DatadogSpansDispatcher(WebClient datadogClient,
                                   ObjectMapper mapper,
-                                  DatadogForwarderConfigProperties properties) {
+                                  DatadogForwarderConfigProperties properties,
+                                  MetersProvider metersProvider) {
         this.datadogClient = datadogClient;
         this.mapper = mapper.writer();
         this.pending = new ArrayBlockingQueue<>(properties.getQueuedMaxSpans());
+        this.successCounter = metersProvider.forwarderCounter("datadog", true);
+        this.failureCounter = metersProvider.forwarderCounter("datadog", false);
```

Comment:
I'm surely missing knowledge here, but I don't see anything in the counter's name that makes me understand that this counter's semantics is that of a success or a failure. The name is just "datadog" with value "true" or "false", I was expecting something like "datadog success" or similar?

Reply:
The boolean is the `success` tag.

The metrics look like this:
`counter.pitchfork.something{forwarder=datadog, success=true}`
or
 `counter.pitchfork.something{forwarder=datadog, success=false}`

Reply:
Perfect thanks!